protocols/grpc/agent.proto

```diff
@@ -266,6 +266,7 @@ message CreateSandboxRequest {
 	// one sandbox per agent and implicitly require that CreateSandbox is
 	// called before other sandbox/network calls.
 	string sandbox_id = 5;
+	string guest_hook_path = 6;
```

Comment:
Please can you add a comment here explaining what this field is. Also, please specify if the path is expected to be an absolute one.

Related: #150.